project/cmake/CMakeLists.txt

```diff
@@ -331,7 +331,7 @@ else()
 endif()
 add_dependencies(${APP_NAME_LC} ${APP_NAME_LC}-libraries export-files pack-skins)
 whole_archive(_MAIN_LIBRARIES ${core_DEPENDS})
-target_link_libraries(${APP_NAME_LC} ${_MAIN_LIBRARIES} lib${APP_NAME_LC})
+target_link_libraries(${APP_NAME_LC} ${_MAIN_LIBRARIES} lib${APP_NAME_LC} ${DEPLIBS})
```

Comment:
Sry, i don't understand it from reading the commit message. 3eb4687 was fixing the issue that libs were specified twice in the commit message. How should we then deal with it?